package.json

```diff
@@ -1,16 +1,32 @@
 {
+  "name": "vitamin-web",
   "private": true,
+  "keywords": [
+    "design-system",
+    "ui-kit",
+    "libraries"
+  ],
+  "bugs": {
+    "url": "https://github.com/Decathlon/vitamin-web/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Decathlon/vitamin-web.git"
+  },
+  "license": "Apache-2.0",
+  "workspaces": [
+    "packages/*"
+  ],
```

Comment:
Because you propose to use Yarn, we should add `engines` property inside `package.json` to force the user to have Yarn installed (https://classic.yarnpkg.com/en/docs/package-json/#toc-engines)
Also, we need to add this dependency and how to install's link inside our `README.md`